safe-smart-account/supported-networks/v1.3.0.md

```diff
@@ -2220,22 +2220,22 @@ This network's chain ID is 222000222.
 - `simulate_tx_accessor.sol`: [0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da](https://subnets-test.avax.network/meld/address/0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da)
 
 
-### Neon EVM DevNet
+### Neon EVM Devnet
```

Comment:
@louis-md Can you add this to the Vale dictionary and remove the other one if its not correct?